texts/en_US.properties

```diff
@@ -172,8 +172,12 @@ geyser.scoreboard.updater.threshold_reached=We will decrease the amount that the
 geyser.scoreboard.updater.threshold_reached.log={0} received more than {1} score packets per second (currently {2}).
 geyser.scoreboard.updater.threshold_reached.player=You received more than {0} score packets per second (currently {1}).
 
-geyser.settings.option.coordinates=Show coordinates
 geyser.settings.title.client=Client Settings
+geyser.settings.option.coordinates=Show coordinates
+geyser.settings.option.cooldown_animation=Attack Cooldown Animation
+geyser.settings.option.cooldown_animation.title=Under Crosshair
```

Comment:
Anyone got ideas for something other than `geyser.settings.option.cooldown_animation.title`? I don't like it that much because `title` is already a key for the actual titles in the form. Maybe `geyser.settings.option.cooldown_animation.crosshair` ?

Reply:
closed because of 27c76d43eebacfc558a5add6a451d42aa4f601df